build.gradle

```diff
@@ -1,102 +1,111 @@
-plugins {
-    id "java"
-    id "edu.wpi.first.GradleRIO" version "2020.1.2"
-}
-
-sourceCompatibility = JavaVersion.VERSION_13
-targetCompatibility = JavaVersion.VERSION_13
-
-def ROBOT_MAIN_CLASS = "frc.robot.Main"
-

```

Comment:
Revert this file

Reply:
Got moved, and we need to go with java 11 until they update the rio